### PR TITLE
Fix missing completions for object constructor expression and panic statement [2201.2.x]

### LINKED ISSUE #37225: Improve completions in panic statements
**Description:**
$Title

https://user-images.githubusercontent.com/46120162/182382766-68a4ad0a-2699-4821-813e-59552c948e2c.mov

**Describe your problem(s) and solutions(s)**
1. We have a snippet for panic completions but it just provide the edit `panic `. We can provide `panic error("")` if we are providing a snippet since we are expecting an error after panic keyword.
2. We can add another completion with `panic` keyword as a keyword completion. 
3. Instead of `error` keyword, we should also provide `error` constructor completion along with `error` keyword for ease of use.

### LINKED ISSUE #37453: Missing init function completion inside object constructor expression
**Description:**
Screenshot 2022-08-18 at 17 17 19

**Steps to reproduce:**

**Affected Versions:**
2201.2.0

### PR DESCRIPTION
**Pull request by gayalkuruppu**
Fixes #37453 
Fixes #37225 
Fixes https://github.com/ballerina-platform/ballerina-lang/issues/37687

## Check List 
- [x] Read the [Contributing Guide](https://github.com/ballerina-platform/ballerina-lang/blob/master/CONTRIBUTING.md)
- [ ] Updated Change Log
- [ ] Checked Tooling Support (#<Issue Number>)
- [x] Added necessary tests
  - [x] Unit Tests
  - [ ] Spec Conformance Tests
  - [ ] Integration Tests
  - [ ] Ballerina By Example Tests
- [x] Increased Test Coverage   
- [ ] Added necessary documentation  
  - [ ] API documentation 
  - [ ] Module documentation in Module.md files
  - [ ] Ballerina By Examples

### CODE REVIEW

**Review by malinthar — DISMISSED** [review]
LGTM!

File: language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/ObjectBodiedNodeContextProvider.java [hunk 1]
```diff
@@ -86,22 +85,22 @@ protected List<LSCompletionItem> getBodyContextItems(BallerinaCompletionContext
...
                 && ((ObjectConstructorExpressionNode) node).objectTypeQualifiers().stream()
-                .anyMatch(token -> token.kind() == SyntaxKind.SERVICE_KEYWORD));
-    }
-
-    private boolean isClientObject(Node node) {
-        return (node.kind() == SyntaxKind.OBJECT_CONSTRUCTOR
-                && ((ObjectConstructorExpressionNode) node).objectTypeQualifiers().stream()
-                .anyMatch(token -> token.kind() == SyntaxKind.CLIENT_KEYWORD));
+                .anyMatch(token -> token.kind() == SyntaxKind.CLIENT_KEYWORD
+                        || token.kind() == SyntaxKind.CLIENT_KEYWORD));
```
[the remark this answers is not in the document]
Comment (gayalkuruppu) [reply language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/ObjectBodiedNodeContextProvider.java]: Fixed in cd39bc964a2ebbafa38787f5ccfa1ed11da5b71e

**Comment by gayalkuruppu** [comment]
A code action test (`test[70](createVariableWithCheck6.json)`, `org.ballerinalang.langserver.codeaction.CreateVariableTest`) is failing. Changes are only on completions. @malinthar @TharushiJay  any thoughts?

**Comment by IMS94** [comment]
Closing this since we decided to send this only to master and 2201.3.x

**Pull request closed without merging**